mava/systems/tf/mappo/training.py

```diff
@@ -536,7 +642,60 @@ def __init__(
+            agent_types :  agent types, e.g. "speaker" or "listener".
+            observation_networks : observation networks for each agent in the system.
+            policy_networks : policy networks for each agent in the system.
+            critic_networks : critic network(s), shared or for each agent in the system.
+            dataset : training dataset.
+            policy_optimizer : optimizer for updating policy networks.
+            critic_optimizer : optimizer for updating critic networks. This is not
+                necessary if using single optim.
+            use_single_optimizer : boolean to decide
+                whether or not the critic, policy and observation networks are
+                optimized jointly by a single optimizer. If true, all networks
+                are optimized by the policy_optimizer. If False, the observation and
+                policy network are optimized by the policy optimizer and the
+                critic network by the critic optimizer.
+            agent_net_keys : specifies what network each agent uses.
+            checkpoint_minute_interval : The number of minutes to wait between
+                checkpoints.
+            minibatch_size : size of minibatch that is sampled from
+                the training batch. Minibatches are used for each gradient step.
+            num_epochs : number of epochs for every training step.
+                Recommendation as per https://arxiv.org/pdf/2103.01955.pdf, "15
+                epochs for easy tasks,and 10 or 5 epochs for difficult tasks."
+            discount :  discount factor for TD updates.
+            lambda_gae : scalar determining the mix of bootstrapping
+                vs further accumulation of multi-step returns at each timestep.
+            entropy_cost : contribution of entropy regularization to
+                the total loss. Defaults to 0.0.
+            baseline_cost : contribution of the value loss to the
+                total loss. Defaults to 1.0.
+            clipping_epsilon : Hyper-parameter for clipping in the
+                policy objective. Defaults to 0.2.
+            max_gradient_norm : maximum allowed norm for gradients
+                before clipping is applied. Defaults to None.
+            counter : step counter object. Defaults to None.
+            logger : logger object for logging trainer
+                statistics. Defaults to None.
+            checkpoint : whether to checkpoint networks. Defaults to
+                True.
+            checkpoint_subpath : subdirectory for storing checkpoints.
+                Defaults to "~/mava/".
+            learning_rate_scheduler_fn: dict with two functions (one for the policy and
+                one for the critic optimizer), that takes in a trainer step t and
+                returns the current learning rate.
+            normalize_advantage: whether to normalize the advantage.
+            value_clipping: whether to clip the difference
```

Comment:
As mentioned in another comment, I think we should maybe remove this.

Reply:
Removed :+1:

Reply:
Thanks